### PR TITLE
[IMP] theme_paptic: update the theme

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Update the theme to improve the design with shapes, new pictures and new custom snippets.

**Improvements :**
- Update the tour : add Comparisons and Call_to_action
- Cover : change color preset to improve the color of buttons
- Image_text : add more space before the arrow of the button
- Three_columns : improve the layout of cards and keep the original card structure
- Update the screenshot
- Picture : change the shape color for a better contrast with the title.

**New customisations :**
- Comparisons
- Call_to_action
- Banner
- Numbers
- Big Boxes (Color_blocks_2)
- Features
- Media_list
- Features_grid
- Pricelist

Task-2602618

### CODE REVIEW

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
I just flied over the PR, nothing too scary I think :+1: (obviously the less custo I see, the better I'll sleep but... :laughing:)

Make sure to take care of my comments (for all of your snippets and other themes too, and share the comments with the other designers :+1: )

Note: commit message:
- This commit update**s**
- Max 72 characters per line in the description but use those 72 characters (not 50 like you did)

Courage ! :)

File: theme_paptic/data/ir_asset.xml [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
-        </record>
-
-    </data>
+    <record id="theme_paptic.bootstrap_overridden_scss" model="theme.ir.asset">
+        <field name="key">theme_paptic.bootstrap_overridden_scss</field>
+        <field name="name">Bootstrap overridden SCSS</field>
+        <field name="bundle">web._assets_frontend_helpers</field>
+        <field name="directive">prepend</field>
+        <field name="path">theme_paptic/static/src/scss/bootstrap_overridden.scss</field>
+    </record>
```
Comment (qsm-odoo) [inline theme_paptic/data/ir_asset.xml:16]: When you reindent, reindent following our convention, no indentation and blank line around:

```xml
<odoo>

<record>
[...]
</record>

</odoo>
```

File: theme_paptic/static/src/js/tour.js [hunk 2]
```diff
@@ -1,11 +1,5 @@
-odoo.define("theme_paptic.tour.paptic", function (require) {
-"use strict";
-
-const core = require("web.core");
-const _t = core._t;
-
-const wTourUtils = require("website.tour_utils");
-var tour = require("web_tour.tour");
+/** @odoo-module */
+import wTourUtils from 'website.tour_utils';
```
Comment (qsm-odoo) [inline theme_paptic/static/src/js/tour.js:2]: I know I am annoying but if you can add a blank line after `/** @odoo-module */`... thanks :+1:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
@Brieuc-brd This seems good :+1: I only have two useless comments to make, if you can handle them and make sure no other change is needed, I'll then test and merge this one today :+1:

Note: I already rebased on 14.5, push-forced and retargeted the PR.

Could you also adapt (if needed) the related IAP PR and mention it in the paptic task? So we can merge them together. Thanks :+1:

**Comment by Brieuc-brd** [comment]
Hello @qsm-odoo 👋 
I made the changes you asked and added the link of the IAP PR in the paptic task. 👍 
I also add it here : [https://github.com/odoo/iap-apps/pull/404](url)

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Comment by qsm-odoo** [comment]
@robodoo r-

**Comment by qsm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**